pkg/porter/schema/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+    "type": "object",
+    "properties": {
+      "name": {
+        "type": "string"
+      },
+      "description": {
+        "type": "string"
+      },
+      "version": {
+        "type": "string"
+      },
+      "image": {
```

Comment:
Should this be `invocationImage`?  (Referencing the two `porter.yaml` examples under `examples`) . Or would that  fall under `additionalProperties`?

Speaking of which, what is the delineation of properties that are 'top level' and that are grouped under `additionalProperties`?

Reply:
@vdice `additionalProperties: false/true` simply means "do/don't error if you see properties other than those listed".  It's not parking a second set of properties away somewhere; it's saying "it's okay if the document contains other properties too".

Reply:
oops! Thanks I accidentally looked at the struct field instead of the yaml. 🤦‍♀️ 

There shouldn't be anything under additional properties. I just have it turned on for now because it's not the full schema while we test this out with Ivan and VS Code. Once we have the full schema in place, we'll turn it to false.